test/behave/12_tracker.feature

```diff
@@ -4,3 +4,18 @@ Feature: tracker
     Given an English speaking user
     When the user says "give me location of the mark"
 	  Then mycroft reply should contain "home"
+
+  Scenario: tracker - where skill conflict #1
+    Given an English speaking user
+    When the user says "give me location of the Czech Republic"
+	  Then "skill-homeassistant" should not reply
+
+  Scenario: tracker - where skill conflict #2
+    Given an English speaking user
+    When the user says "where is France"
+	  Then "skill-homeassistant" should not reply
+
+  Scenario: tracker - where skill conflict #3
+    Given an English speaking user
+    When the user says "where is the golden gate bridge"
+	  Then "skill-homeassistant" should not reply
```

Comment:
Optional - where you want to re-use a Scenario, you can use a Scenario Outline:
https://mycroft-ai.gitbook.io/docs/skill-development/voight-kampff/scenario-outlines

Reply:
Perfect, I tried to find a way to add more examples but I did not know correct naming of it.